### ISSUE TITLE
Add Code Coverage to README.md

### ISSUE THREAD

**Issue by wpank** [sourcecred/sourcecred#1069]
High code coverage is a good sign of a healthy project. 

Running `yarn coverage` gives us the following:
```
------------------------------------|----------|----------|----------|----------|-------------------|
File                                |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------|----------|----------|----------|----------|-------------------|
All files                           |    80.39 |    75.15 |    83.43 |    84.32 |                   |
```

This relatively high code coverage wouldn't be known to anyone besides those that took the time to set up the project and run the above command. 

Including it in the projects README.md is a good way to publicly and transparently showcase code coverage. 

### Proposed Implementation
[Node-Coveralls](https://github.com/nickmerwin/node-coveralls) seems to be a popular option for node.js projects. 

[This article describes how to setup Coveralls with CircleCI and Jest](https://zach.codes/ci-code-coverage-tests-in-5-minutes/)

This involves:
- Installing the Coveralls module
- Signing up for Coveralls
- Creating a Coveralls test command
- Updating the CircleCI config file
- Creating a COVERALLS_REPO_TOKEN env var in CircleCI

**Comment by decentralion** [comment]
I support this, and I suspect @wchargin would as well!

**Comment by wchargin** [comment]
Yep, we’ve discussed this in the past and been in favor of doing it, but
just never bothered.

In SourceCred, our Jest tests and Sharness tests complement each other.
It would be awesome if the coverage report could take into account the
coverage induced by Sharness tests, but I appreciate that this might
require some wiring, so I’d be happy to see just the basic reuslts of
`yarn coverage` as a first step.

I’ve used Coveralls in the past and have been happy with it, although it
gets really confused if you rename a repository (or at least it used
to).

> Creating a COVERALLS_REPO_TOKEN env var in CircleCI

It may suffice to run `coveralls <./coverage/lcov.info`. The `coveralls`
binary autodetects popular CI systems, including Travis and CircleCI.

> Updating the CircleCI config file

Yep, I think that it makes sense to add this to the CI file specifically
rather than a conditional step in `yarn test` (but could be convinced
otherwise).
